layouts/community/ortho_4x12/xyverz/config.h

```diff
@@ -0,0 +1,30 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#define MIDI_BASIC
+
+#define USE_SERIAL
+
+#define EE_HANDS
+
+#undef RGBLED_NUM
+#ifdef KEYBOARD_planck_rev5
+  #define RGBLED_NUM 10
+  #define RGB_DI_PIN D1
+#endif
+#ifdef KEYBOARD_levinson
```

Comment:
Until it's fixed, you'd want to use `KEYBOARD_levinson_rev2` here (or `KEYBOARD_levinson_rev1`).  Eg, the full keyboard name.